pkgs/development/python-modules/slovnet/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+, razdel
+, navec
+, pytestCheckHook
+}: buildPythonPackage rec {
+  pname = "slovnet";
+  version = "0.6.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-AtIle9ybnMHSQr007iyGHGSPcIPveJj+FGirzDge95k=";
+  };
+
+  propagatedBuildInputs = [ numpy navec razdel ];
+  nativeCheckInputs = [ pytestCheckHook ];
+  pytestFlagsArray = [ "tests/" ];
+  disabledTestPaths = [ "tests/test_api.py" ];
```

Comment:
ditto

Reply:
I missed this.
> Some tests disabled due to sandbox unfriendly tests scripts, e.g. slovnet trying to download model binary artifacts.
```suggestion
  disabledTestPaths = [ 
    # tries to download model binary artifacts
    "tests/test_api.py"
  ];
```

Reply:
I've added this comment into `slovnet` commit. Thanx!